comprobo_final_project/scripts/main.py

```diff
@@ -35,9 +36,10 @@ def run(self):
         generation = 0
         found = False
         while generation < self.maxGenerations:
-            print"Generation %d: %s" % (generation, self.population.population[0].genes), self.population.population[0].fitness
+            print"Generation %d: %s" % (generation, self.population.generation[0].genes), self.population.generation[0].fitness
+
             if self.population.population[0].fitness < 0.05:
-                print "Most fit gene:", self.population.population[0].genes, self.population.population[0].fitness
+                print "Most fit gene:", self.population.generation[0].genes, self.population.generation[0].fitness
```

Comment:
each "population" though is a single generation, which is why I called it that. it's also consistent with the variable name in population.py